examples/persistent-chat.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use PhpLlm\LlmChain\Chain\Chain;
+use PhpLlm\LlmChain\Chain\Chat;
+use PhpLlm\LlmChain\Chain\Chat\MessageStore\InMemoryStore;
+use PhpLlm\LlmChain\Platform\Bridge\OpenAI\GPT;
+use PhpLlm\LlmChain\Platform\Bridge\OpenAI\PlatformFactory;
+use PhpLlm\LlmChain\Platform\Message\Message;
+use PhpLlm\LlmChain\Platform\Message\MessageBag;
+use Symfony\Component\Dotenv\Dotenv;
+
+require_once dirname(__DIR__).'/vendor/autoload.php';
+(new Dotenv())->loadEnv(dirname(__DIR__).'/.env');
+
+if (empty($_ENV['OPENAI_API_KEY'])) {
+    echo 'Please set the OPENAI_API_KEY environment variable.'.\PHP_EOL;
+    exit(1);
+}
+
+$platform = PlatformFactory::create($_ENV['OPENAI_API_KEY']);
+$llm = new GPT(GPT::GPT_4O_MINI);
+
+$chain = new Chain($platform, $llm);
+$chat = new Chat($chain, new InMemoryStore());
+
+$messages = new MessageBag(
+    Message::forSystem('You are a helpful assistant. You only answer with short sentences.'),
+);
+
+$chat->start($messages);
```

Comment:
```suggestion
$chat->initialize($messages);
```
? Bootstrap? Configure?

Reply:
initiate is nice

Reply:
`initiate` it is :+1: